### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found using command below:

    codespell . -S *.js,*.ts,./playwright,./driver,./.eggs  -L complies,datas,keypair

### CODE REVIEW

**Comment by rwoll** [comment]
Thanks for the PR. The errors in the files under `_generated` come from upstream (https://github.com/microsoft/playwright/tree/main/docs/src). Do you want to fix them there and then we can see what's left here?

**Comment by kianmeng** [comment]
> Thanks for the PR. The errors in the files under `_generated` come from upstream (https://github.com/microsoft/playwright/tree/main/docs/src). Do you want to fix them there and then we can see what's left here?

Noted. See https://github.com/microsoft/playwright/pull/16370.

**Comment by rwoll** [comment]
Thanks! Once https://github.com/microsoft/playwright-python/pull/1485 lands, you can rebase on that.

File: playwright/async_api/_generated.py [hunk 1]
```diff
@@ -9263,7 +9263,7 @@ async def pause(self) -> NoneType:
         User can inspect selectors or perform manual steps while paused. Resume will continue running the original script from
         the place it was paused.
 
-        > NOTE: This method requires Playwright to be started in a headed mode, with a falsy `headless` value in the
+        > NOTE: This method requires Playwright to be started in a headed mode, with a falsely `headless` value in the
```
Comment (mxschmitt) [inline playwright/async_api/_generated.py:9266]:
```suggestion
        > NOTE: This method requires Playwright to be started in a headed mode, with a falsy `headless` value in the
```

File: playwright/sync_api/_generated.py [hunk 2]
```diff
@@ -9332,7 +9332,7 @@ def pause(self) -> NoneType:
         User can inspect selectors or perform manual steps while paused. Resume will continue running the original script from
         the place it was paused.
 
-        > NOTE: This method requires Playwright to be started in a headed mode, with a falsy `headless` value in the
+        > NOTE: This method requires Playwright to be started in a headed mode, with a falsely `headless` value in the
```
Comment (mxschmitt) [inline playwright/sync_api/_generated.py:9335]:
```suggestion
        > NOTE: This method requires Playwright to be started in a headed mode, with a falsy `headless` value in the
```

**Review by mxschmitt — APPROVED** [review]

**Comment by rwoll** [comment]
Landed! Thanks @kianmeng

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**